astroid/rebuilder.py

```diff
@@ -1354,7 +1364,8 @@ def visit_attribute(
             newnode = nodes.AssignAttr(node.attr, node.lineno, node.col_offset, parent)
             # Prohibit a local save if we are in an ExceptHandler.
             if not isinstance(parent, nodes.ExceptHandler):
-                self._delayed_assattr.append(newnode)
+                # mypy doesn't recognize that newnode has to be AssignAttr
+                self._delayed_assattr.append(newnode)  # type: ignore
```

Comment:
Only for the moment. `AssignAttr.__init__` has a decorator that uses `ParamSpec` which is unsupported currently.
I believe this will also be fixed with the next release.